components/Elevation/src/UIColor+MaterialElevation.m

```diff
@@ -48,10 +48,20 @@ - (UIColor *)mdc_resolvedColorWithElevation:(CGFloat)elevation {
   elevation = MAX(elevation, 0);
   CGFloat alphaValue = 0;
   if (!MDCCGFloatEqual(elevation, 0)) {
-    // A formula is used here to simulate the alpha percentage stated on
-    // https://material.io/design/color/dark-theme.html#properties
-    // AlphaValue = 4.5 * ln (elevationValue + 1) + 2
-    alphaValue = (CGFloat)4.5 * (CGFloat)log(elevation + 1) + 2;
+    if (elevation < 1) {
+      // A formula for values between 0 to 1 is used here to simulate the alpha percentage
+      // as in the main formula below there is a jump between any number larger than 0 to an
+      // alpha value of 2. This formula provides a gradual polynomial curve that makes the delta
+      // of the alpha value between lower numbers to be smaller than the higher numbers.
+      // AlphaValue = 5.11916 * elevationValue ^ 2
+      alphaValue = (CGFloat)5.11916 * MDCPow((CGFloat)elevation, 2);
```

Comment:
I believe this should be closer to 3.35463. In the below formula if I'm doing the order of operations correctly it comes out to this number. I believe at 1, it would be as follows.
1. Log(2) = ~0.3
2. 4.5 * 0.3 = ~1.35
3. 2 + 1.35 = ~3.35

That would mean that this value should be ~3.35 * MDCPow(elevation, 2)

Reply:
the `log` function in iOS is with the base of e and not 10

Reply:
Whoops I was using `log10` not `log`.

Reply:
@yarneo Yeah just found that out, thanks!